stanford/roles/cluster/defaults/main.yml

```diff
@@ -7,10 +7,10 @@ CLUSTER_SECURITY_GROUP: "{{ COMMON_DEPLOYMENT }}-group-{{ CLUSTER_NAME }}"
 CLUSTER_SECURITY_GROUP_DESCRIPTION: "Secure {{ COMMON_DEPLOYMENT }}-{{ CLUSTER_NAME }} clusters"
 CLUSTER_SUBNET_AZ: "{{ region }}a"
 CLUSTER_SUBNET_TAGS:
-      Name: "{{ COMMON_DEPLOYMENT }}-subnet-{{ CLUSTER_NAME }}-{{ CLUSTER_NUMBER }}"
+      Name: "{{ COMMON_DEPLOYMENT }}-subnet-{{ CLUSTER_NAME }}-{{ DEPLOYMENT_NUMBER }}"
```

Comment:
These two changes here, `s/CLUSTER_NUMBER/DEPLOYMENT_NUMBER/`, fix a bug that affects nodes with `CLUSTER_NUMBER > 2 or CLUSTER_NUMBER <= 0`.

We only ever have two subnets per cluster (numbered 1 and 2, respectively), no matter the instance number, so we need to use `DEPLOYMENT_NUMBER` instead of `CLUSTER_NUMBER`.

Else, if you're deploying say `CLUSTER_NUMBER=0` for privateprod, this would have mistakenly renamed this subnet from `lagunita-subnet-edxapp-2` to `lagunita-subnet-edxapp-0`, which isn't what we want.